### DISCUSSION TITLE
Warning About Downgrading + Unenrolling (Laws + Punishments) for you 12 yr olds.

### DISCUSSION THREAD

**Discussion by Alex2Mayhem** [3kh0/ext-remover#187] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Okay for you people who have no idea what your doing and why you shouldn't do it.  and the kids that are complaining
 Read this below

Even first-time offenses for accessing a protected computer without sufficient "authorization" can be punishable by up to five years in prison each (ten years for repeat offenses), plus fines. Violations of other parts of the CFAA are punishable by up to ten years, 20 years, and even life in prison.

Goverment Law.

SUMMARY

Someone who hacks into another person's computer could be punished by a number of different crimes, depending on the circumstances. The law punishes hacking under the computer crime statutes. These crimes carry penalties ranging from a class B misdemeanor (punishable by up to six months in prison, a fine of up to $1,000, or both) to a class B felony (punishable by up to 20 years in prison, a fine of up to $15,000, or both). The law also punishes unauthorized access to a computer or computer network, with penalties ranging from a class B misdemeanor to a class D felony (punishable by up to five years in prison, a fine of up to $5,000, or both).

A number of generally applicable crimes could also apply. For example, hacking could be done to commit identity theft or larceny and it could be punished under those generally applicable crimes.
In addition to criminal penalties, the law specifically authorizes someone harmed by a computer or unauthorized use crime to bring a civil lawsuit against the perpetrator. These civil actions are in addition to any other grounds for a civil action that the injured party may have.

COMPUTER CRIMES

A person commits a “computer crime” when he or she:

1. accesses a computer system without authorization;

2. accesses or uses a computer system to obtain unauthorized computer services (including computer access, data processing, and data storage);

3. intentionally or recklessly disrupts, degrades, or causes disruption or degradation of computer services or denies or causes denial of computer services to an authorized user; or

4. intentionally or recklessly tampers with, takes, transfers, conceals, alters, or damages any equipment used in a computer system.
It is also a computer crime to misuse computer system data. A person commits this crime by:

1. accessing a computer system to use, disclose, or copy data residing in, communicated by, or produced by a computer system;

2. intentionally or recklessly and without authorization (a) tampering with, damaging, or taking data intended for use by a computer system or (b) intercepting or adding to data residing within a computer system;

3. knowingly receiving or retaining data obtained through misuse of computer system information; or

4. using or disclosing data he or she knows or believes was obtained through misuse of computer system information (CGS § 53a-251).

The punishment for committing one of these computer crimes depends on the damage caused and risk of harm created. Table 1 displays the five degrees of computer crime, the amount of damage or harm required for each, and their penalties.

This may vary for different states.

Please don't be dumb you're risking yourself. Your grades, and your Education.

**Comment by synapt0** [comment]
Tbh if they want to do it that bad, they should just get a new Chromebook without enrollment. I'll just wait for the next update.

**Comment by Sxri** [comment]
good thing i live in ohio.

**Reply by ghost** [reply]
Down in ohio Down in ohio

**Reply by Fallenmoon8080** [reply]
Real L moment living in ohio. Still don't downgrade bro

**Reply by Sxri** [reply]
> Real L moment living in ohio. Still don't downgrade bro

I was kidding, I'm not an idiot. I play it safe don't worry, I've been down this path before.

**Reply by ghost** [reply]
we all have...

**Reply by cooltoo37** [reply]
i downgraded to 105 so i could use compact cow ui and now i just got sh1mmer totally gonna go to jail now  dayum i am hella stupid well i could just say i lost it and go through a 300 fine instead of going through law shit

**Reply by Nintendoboi22** [reply]
me waiting for the police to show up at my door after i free my school of iboss and gogaurdian

**Reply by SlitheringLizard1** [reply]
bro I fake everything when I break my Chromebook. I play all innocent like I have no idea what I did, like I know nothing about these Chromebooks. I might know more than the person who gives me a new chromebook when I accidentally wipe all wifi data from my chromebook
imma go to prison aren't I

**Comment by Terrinn** [comment]
Wait who...

**Reply by ghost** [reply]
everyone

**Comment by jonahnm** [comment]
Thank you for writing this warning, I sincerely hope kids actually listen.

**Reply by ghost** [reply]
Well, some weren't

**Reply by Alex2Mayhem** [reply]
> Thank you for writing this warning, I sincerely hope kids actually listen.

Your welcome

**Reply by ghost** [reply]
My friend's younger (13 year old) brother downgraded his chromebook, he got suspended for 2 weeks and had detension every lunch and brunch for a week after that. Parents had to meet with district. Pretty sure he ruined his whole education from that.

**Reply by 3kh0** [reply]
@WizardCraft112 L bozo, he did it wrong most likely.

**Reply by ghost** [reply]
exactly

**Reply by ghost** [reply]
@3kh0 Doing it at all is wrong.

**Reply by ghost** [reply]
yea, the only reason he wanted to do it was because he lied to his teacher and his parents grounded him for a month so he couldn't play video games :/

**Reply by cooltoo37** [reply]
bro for me its been like 6 months downgraded my chromebook to 105 and havent got caught yet i just need some training anybody wanna teach me i need to learn java script first pls

**Comment by funkymokey** [comment]
Am I allowed to do this if I have received a "personal" Chromebook with enrollment on it?

**Reply by qDread-dev** [reply]
Why is personal in quotes? If your district assigned chromebooks 1:1 then no.

**Reply by rwg100** [reply]
if you bought the chromebook and the district didnt give it to you and you own it, then i think its fine to use sh1mmer.

**Reply by rwg100** [reply]
just dont use sh1mmer if the school gave you the chromebook

**Reply by 3kh0** [reply]
yes you can

**Reply by testacc992** [reply]
12 year olds 👍 
-roka

**Reply by funkymokey** [reply]
> Why is personal in quotes? If your district assigned chromebooks 1:1 then no.

the chromebook was given to me, not by a school though the reason i said personal in quotes is because its enrolled

**Reply by qDread-dev** [reply]
> > Why is personal in quotes? If your district assigned chromebooks 1:1 then no.
> 
> the chromebook was given to me, not by a school though the reason i said personal in quotes is because its enrolled

Who gave it to you then?

**Reply by BroJac5246** [reply]
> > Why is personal in quotes? If your district assigned chromebooks 1:1 then no.
> 
> the chromebook was given to me, not by a school though the reason i said personal in quotes is because its enrolled

Yes, you can, but be warned that it is likely a stolen device. I'd get in contact with the district first to see what's going on.

**Reply by funkymokey** [reply]
It was given to me with the enrollment hcpl.net, i have already removed the enrollment

**Reply by obkduishinijf** [reply]
> It was given to me with the enrollment hcpl.net, i have already removed the enrollment

harris county public library, idk where you are but it's probably stolen.

**Reply by BroJac5246** [reply]
> It was given to me with the enrollment hcpl.net, i have already removed the enrollment

Please contact them. It would be really immoral to have a stolen device from a library, and at the very least they should be aware what happened. You can report it anonymously, just please report it and work it out with them.

**Reply by sillyangel** [reply]
its not stolen, it is a program that they run ("righting this comment on one") here is a link [https://www.hcpl.net/services/connected](https://www.hcpl.net/services/connected)

**Reply by BroJac5246** [reply]
> its not stolen, it is a program that they run ("righting this comment on one") here is a link [https://www.hcpl.net/services/connected](https://www.hcpl.net/services/connected)

Considering it's managed in the first place, I'd assume they want it to stay managed, but whatever 🤷‍♀️

**Reply by funkymokey** [reply]
I know this is dead but I really should have said that it was given to me from hpcl.net (library) and that it was managed even though there was no restriction to the internet (except for apps)

**Reply by BroJac5246** [reply]
> I know this is dead but I really should have said that it was given to me from hpcl.net (library) and that it was managed even though there was no restriction to the internet (except for apps) 

It sounds like they want it to stay enrolled, but I'd say it comes down to whether you have to give it back at some point. Ultimately, it's probably okay, though, unless they have a reason for having it enrolled.

**Reply by bartholemuew** [reply]
> It sounds like they want it to stay enrolled

skill issue ngl

**Reply by sillyangel** [reply]
@funkymokey how did you get ur shim and what board do u have??

**Comment by amelia-bartlett** [comment]
> Okay for you people who have no idea what your doing and why you shouldn't do it. and the kids that are complaining Read this below
> 
> Even first-time offenses for accessing a protected computer without sufficient "authorization" can be punishable by up to five years in prison each (ten years for repeat offenses), plus fines. Violations of other parts of the CFAA are punishable by up to ten years, 20 years, and even life in prison.
> 
> Goverment Law.
> 
> SUMMARY
> 
> Someone who hacks into another person's computer could be punished by a number of different crimes, depending on the circumstances. The law punishes hacking under the computer crime statutes. These crimes carry penalties ranging from a class B misdemeanor (punishable by up to six months in prison, a fine of up to $1,000, or both) to a class B felony (punishable by up to 20 years in prison, a fine of up to $15,000, or both). The law also punishes unauthorized access to a computer or computer network, with penalties ranging from a class B misdemeanor to a class D felony (punishable by up to five years in prison, a fine of up to $5,000, or both).
> 
> A number of generally applicable crimes could also apply. For example, hacking could be done to commit identity theft or larceny and it could be punished under those generally applicable crimes. In addition to criminal penalties, the law specifically authorizes someone harmed by a computer or unauthorized use crime to bring a civil lawsuit against the perpetrator. These civil actions are in addition to any other grounds for a civil action that the injured party may have.
> 
> COMPUTER CRIMES
> 
> A person commits a “computer crime” when he or she:
> 
>     1. accesses a computer system without authorization;
> 
>     2. accesses or uses a computer system to obtain unauthorized computer services (including computer access, data processing, and data storage);
> 
>     3. intentionally or recklessly disrupts, degrades, or causes disruption or degradation of computer services or denies or causes denial of computer services to an authorized user; or
> 
>     4. intentionally or recklessly tampers with, takes, transfers, conceals, alters, or damages any equipment used in a computer system.
>        It is also a computer crime to misuse computer system data. A person commits this crime by:
> 
>     5. accessing a computer system to use, disclose, or copy data residing in, communicated by, or produced by a computer system;
> 
>     6. intentionally or recklessly and without authorization (a) tampering with, damaging, or taking data intended for use by a computer system or (b) intercepting or adding to data residing within a computer system;
> 
>     7. knowingly receiving or retaining data obtained through misuse of computer system information; or
> 
>     8. using or disclosing data he or she knows or believes was obtained through misuse of computer system information (CGS § 53a-251).
> 
> 
> The punishment for committing one of these computer crimes depends on the damage caused and risk of harm created. Table 1 displays the five degrees of computer crime, the amount of damage or harm required for each, and their penalties.
> 
> This may vary for different states.
> 
> Please don't be dumb you're risking yourself. Your grades, and your Education.

im not from the US, your laws don't apply to me.
quit being presumptive (i checked the relevant legislation, and decided i was ok taking this risk)

**Comment by chickennu99et** [comment]
i mean like, only if your dumb and get caught

**Reply by chickennu99et** [reply]
also "if" i was using it i would just be using it to unblock replit to code

**Reply by chickennu99et** [reply]
and kids where i am dont use it because they unblocked coolmathgames because of how much hacking there was

**Comment by GlizxyBoi** [comment]
> Okay for you people who have no idea what your doing and why you shouldn't do it. and the kids that are complaining Read this below
> 
> Even first-time offenses for accessing a protected computer without sufficient "authorization" can be punishable by up to five years in prison each (ten years for repeat offenses), plus fines. Violations of other parts of the CFAA are punishable by up to ten years, 20 years, and even life in prison.
> 
> Goverment Law.
> 
> SUMMARY
> 
> Someone who hacks into another person's computer could be punished by a number of different crimes, depending on the circumstances. The law punishes hacking under the computer crime statutes. These crimes carry penalties ranging from a class B misdemeanor (punishable by up to six months in prison, a fine of up to $1,000, or both) to a class B felony (punishable by up to 20 years in prison, a fine of up to $15,000, or both). The law also punishes unauthorized access to a computer or computer network, with penalties ranging from a class B misdemeanor to a class D felony (punishable by up to five years in prison, a fine of up to $5,000, or both).
> 
> A number of generally applicable crimes could also apply. For example, hacking could be done to commit identity theft or larceny and it could be punished under those generally applicable crimes. In addition to criminal penalties, the law specifically authorizes someone harmed by a computer or unauthorized use crime to bring a civil lawsuit against the perpetrator. These civil actions are in addition to any other grounds for a civil action that the injured party may have.
> 
> COMPUTER CRIMES
> 
> A person commits a “computer crime” when he or she:
> 
> 1. accesses a computer system without authorization;
> 2. accesses or uses a computer system to obtain unauthorized computer services (including computer access, data processing, and data storage);
> 3. intentionally or recklessly disrupts, degrades, or causes disruption or degradation of computer services or denies or causes denial of computer services to an authorized user; or
> 4. intentionally or recklessly tampers with, takes, transfers, conceals, alters, or damages any equipment used in a computer system.
>    It is also a computer crime to misuse computer system data. A person commits this crime by:
> 5. accessing a computer system to use, disclose, or copy data residing in, communicated by, or produced by a computer system;
> 6. intentionally or recklessly and without authorization (a) tampering with, damaging, or taking data intended for use by a computer system or (b) intercepting or adding to data residing within a computer system;
> 7. knowingly receiving or retaining data obtained through misuse of computer system information; or
> 8. using or disclosing data he or she knows or believes was obtained through misuse of computer system information (CGS § 53a-251).
> 
> The punishment for committing one of these computer crimes depends on the damage caused and risk of harm created. Table 1 displays the five degrees of computer crime, the amount of damage or harm required for each, and their penalties.
> 
> This may vary for different states.
> 
> Please don't be dumb you're risking yourself. Your grades, and your Education.

lucky me i didnt downgrade versions

**Reply by qDread-dev** [reply]
Downgrading versions and unenrolling are very different

**Reply by GlizxyBoi** [reply]
> Downgrading versions and unenrolling are very different

ahh ok

**Comment by deryilz** [comment]


**Reply by piplup7575** [reply]
so high

**Reply by qDread-dev** [reply]
no we can not

**Comment by jessiejs** [comment]
technically I *guess* the school *might* be able to sue you
but will they?
the most they would do is expel you, i sincerely doubt they would go through the effort of suing.

**Reply by BroJac5246** [reply]
They almost certainly wouldn't sue; it's not cost-effective. However, they could definitely expel, which would have **major** consequences for your future

**Reply by jessiejs** [reply]
They probably wouldn't expel, just suspend, but it depends on what school district and who catches you.

**Reply by BroJac5246** [reply]
> They probably wouldn't expel, just suspend, but it depends on what school district and who catches you.

In other words, they could do so

**Reply by jessiejs** [reply]
Yes, they *could*, but an expelled student is one they can't make profit from.

**Reply by BroJac5246** [reply]
> can't make profit from.

Yeah but the large majority of students are in public schools, which don't make profit.

**Comment by ghost** [comment]
I’m 14

**Reply by jessiejs** [reply]
nobody gives a shit <3

**Comment by Aka-but-nice** [comment]
> I’m 14

Seriously, no one here fucking asked bro. Why do you think that's valid?

**Reply by Charlieis1nsan3** [reply]
ong

**Comment by op-max** [comment]
No wonder Sh1mmer creators got in legal troubles with google...

**Reply by amelia-bartlett** [reply]
> No wonder Sh1mmer creators got in legal troubles with google...

[Citation needed]

**Comment by jazzymakesstuff** [comment]
i purposely only use exploits where i can turn lightspeed back on if i need to because i'm a pussy. also wtf did he think posting here would do.

**Comment by Charlieis1nsan3** [comment]
not 12 but I know the  risks that why you have to be knowledgeable about what you are doing, you  have to know how to re-enroll it you have to know how to add kiosk apps back  all of that boring stuff.

**Comment by SlitheringLizard1** [comment]
*year
